azure-pipelines/build-release.yml

```diff
@@ -28,33 +28,33 @@ stages:
           matrix:
             Linux_Py_3_6:
               python.version: '3.6'
-              vmImageName: ubuntu-18.04
+              vmImageName: ubuntu-latest
               uploadCoverage: "false"
 
             Linux_Py_3_7:
               python.version: '3.7'
-              vmImageName: ubuntu-18.04
+              vmImageName: ubuntu-latest
               uploadCoverage: "true"
 
             Windows_Py_3_6:
               python.version: '3.6'
-              vmImageName: windows-2019
+              vmImageName: windows-latest
               uploadCoverage: "false"
 
             Windows_Py_3_7:
               python.version: '3.7'
-              vmImageName: windows-2019
-              uploadCoverage: "true"
+              vmImageName: windows-latest
+              uploadCoverage: "false"
```

Comment:
Shouldn't the coverage be uploaded for multiple host platforms?

Reply:
Afair we wanted that only for `mbed-devices` as it has os specific code. 
@madchutney could you confirm?

Reply:
Yes we  only want to upload on multiple host platform on devices, but since this is devices shouldn't we leave this as `true`?

Reply:
D'oh. 🤦‍♂

Reply:
Fixed https://github.com/ARMmbed/mbed-devices/pull/44/commits/1af6ea1924c56493d65a42bbfa7d0bd7ae840fcd.